manifests/eap/client.pp

```diff
@@ -41,6 +41,11 @@
 #   The destination file path inside the JBoss module to put the SSL
 #   certificate authority file. Defaults to 'cacert.pem'.
 #
+# [*ca_certificates*]
```

Comment:
Should the variable be called something like ssl_ca_certificate_paths to be more consistent?